sentry-samples/sentry-samples-spring-boot-jakarta/README.md

```diff
@@ -0,0 +1,19 @@
+# Sentry Sample Spring Boot
+
+Sample application showing how to use Sentry with [Spring boot](http://spring.io/projects/spring-boot).
```

Comment:
I guess we should mention that it's Spring boot Jakarta or something?

Reply:
Need to check how exactly the duplication stuff works here. Can add if easily possible

Reply:
Ah it's on the samples.

Reply:
Updated